### PR TITLE
Python 3 support

### PR DESCRIPTION
**Pull request by reinhardt**
Refs #33

### CODE REVIEW

File: src/node/ext/ldap/_node.py [hunk 1]
```diff
@@ -334,16 +347,21 @@ def __call__(self):
             self.changed = False
             self._action = None
         deleted = [self[key] for key in self._deleted_children]
-        for node in self.storage.values() + deleted:
+        for node in list(self.storage.values()) + deleted:
```
Comment (jensens) [inline src/node/ext/ldap/_node.py]: here I would prefer `itertools.chain`

File: src/node/ext/ldap/base.py [hunk 2]
```diff
@@ -1,13 +1,17 @@
 # -*- coding: utf-8 -*-
+from __future__ import print_function
```
Comment (jensens) [inline src/node/ext/ldap/base.py]: I dont think this is necessary, `print('foo') works in python 2 OOTB

File: src/node/ext/ldap/filter.rst [hunk 3]
```diff
@@ -90,25 +90,36 @@ objects to LDAP filters.::
...
-    >>> filter
-    LDAPDictFilter(criteria={'cn': 'sepp', 'sn': u'meier\xe4'})
+    >>> filter.criteria['sn'] == u'meier\xe4'
+    True
+    >>> filter.criteria['cn'] == 'sepp'
+    True
 
-    >>> str(filter)
-    '(|(cn=sepp)(sn=meier\xc3\xa4))'
+    >>> str(filter) in ['(|(sn=meierä)(cn=sepp))', '(|(cn=sepp)(sn=meier\xc3\xa4))']
```
Comment (jensens) [inline src/node/ext/ldap/filter.rst]: I can not follow here? why check both 'ä' variants?

Comment (reinhardt) [reply src/node/ext/ldap/filter.rst]: The representation differs in Python 2 vs. 3. We don't care what exactly it is, so we allow both.

File: src/node/ext/ldap/base.py (hunk 2 above)
Comment (reinhardt) [reply src/node/ext/ldap/base.py]: I think you're right. Probably inserted by modernize and missed during cleanup.

**Pull request closed without merging**